src/sundials/sundials_mpi_errors.c

```diff
@@ -0,0 +1,53 @@
+/* -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2022, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------*/
+
+#include <mpi.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <sundials/sundials_core.h>
+#include <sundials/sundials_mpi_types.h>
+#include <sundials/impl/sundials_mpi_errors_impl.h>
+
+static inline char* combineFileAndLine(int line, const char* file)
+{
+  size_t total_str_len = strlen(file) + 6; /* TODO(CJB): need to figure out width of line */
```

Comment:
See my related comment in `src/sundials/sundials_errors.c`
```suggestion
  size_t total_str_len = strlen(file) + 6;
```